tasks/system_probe.py

```diff
@@ -173,18 +176,20 @@ def test(ctx, skip_object_files=False, only_check_bpf_bytes=False):
     """
 
     if not skip_object_files:
-        build_object_files(ctx, install=False)
+        build_object_files(ctx, bundle_ebpf=bundle_ebpf)
 
     pkg = "./pkg/ebpf/... ./pkg/network/..."
 
     # Pass along the PATH env variable to retrieve the go binary path
     path = os.environ['PATH']
 
-    cmd = 'go test -mod={go_mod} -v -tags "{bpf_tag}" {pkg}'
+    cmd = 'go test -mod= -v -tags "{bpf_tag}" {pkg}'
```

Comment:
why this change ?

Reply:
Good catch. It should not have been commited